HacktoberfestProject.Web/Services/GithubService.cs

```diff
@@ -31,7 +31,7 @@ public async Task<List<Models.DTOs.PullRequest>> GetPullRequestsForRepo(string o
 			_logger.LogTrace($"Sending request to Github for pull requests on repositoy: {name}");
 			var prs = await _client.PullRequest.GetAllForRepository(owner, name, new PullRequestRequest() { State = ItemStateFilter.All});
 
-			return prs.Select(pr => new PullRequest(pr.Number, pr.Url)).ToList();
+			return prs.Select(pr => new Models.DTOs.PullRequest(pr.Number, pr.Url)).ToList();
```

Comment:
Is there a namespace conflict on `Models.DTOs`?

Reply:
Yes there was.